Assets/MixedRealityToolkit/InputModule/Prefabs/MotionControllers.prefab

```diff
@@ -1,58 +0,0 @@
-%YAML 1.1
```

Comment:
Did you mean to remove this prefab? Can't see it elsewhere

Reply:
I don't believe we were using it any longer. The attach to controller and motion controller visualization scripts made this obsolete. Also it was missing the script reference because it was removed.

Reply:
Okay, so I went back to look. This got wrapped up into the MixedRealityCameraPartent prefab.  But it's not been deprecated. The Visualizer was the missing script reference on this. I can revert, but not sure if it's really needed if it's already on the camera prefab.

Reply:
Okay, I'll be sure to put it back and fix the script reference.

Reply:
Exactly, I believe this is a child of the MRCameraParent.

Reply:
right, if it's on the MRCameraParent, do we need it as it's own prefab?